app/config.py

```diff
@@ -638,9 +637,14 @@ class Staging(Production):
     NOTIFY_ENVIRONMENT = "staging"
 
 
+class Scratch(Production):
+    NOTIFY_ENVIRONMENT = "scratch"
```

Comment:
Ah yeah, makes a lot of sense. We can go over the expected config tomorrow morning.